build.sbt

```diff
@@ -333,9 +333,6 @@ def runLinters(mode: String) = WorkflowStep.Use(
   name = Some(s"Run linters in $mode mode"),
   params = Map(
     "github_token"         -> "${{ secrets.GITHUB_TOKEN }}",
-    "auto_fix"             -> "true",
```

Comment:
So, this means that it won't add the commit to fix linting errors? Will the build fail, instead?

Reply:
Yes, the auto fix unfortunately seems flaky. You'd need to fix the errors locally, like with `scalafmt`